packages/trip-details/package.json

```diff
@@ -11,7 +11,7 @@
   "license": "MIT",
   "private": false,
   "dependencies": {
-    "@opentripplanner/core-utils": "^11.4.0",
+    "@opentripplanner/core-utils": "^11.4.1",
```

Comment:
Note that this is to accommodate tests only, and that no release will be triggered for this package.